core/deployment/src/main/java/io/quarkus/deployment/builditem/TipOfTheDayBuildItem.java

```diff
@@ -0,0 +1,18 @@
+package io.quarkus.deployment.builditem;
+
+import org.wildfly.common.Assert;
```

Comment:
:jedi: this is not the `Assert` you are looking for.

Reply:
That one is being used in other build items. I'll send a separate PR for that.